### PR TITLE
fix: add none of the above to allergens

### PR DESCRIPTION
**Pull request by winniezheng**

### CODE REVIEW

**Review by GitAndrew — APPROVED** [review]

File: src/utils/constants.ts [hunk 1]
```diff
@@ -16,4 +16,5 @@ export const allergens = [
   "Fish",
   "Lupin",
   "Soy",
+  "None of the above",
```
Comment (chrisrabe) [inline src/utils/constants.ts:19]: How come? Does it not allow you to go next screen when nothing's selected? :O

Comment (chrisrabe) [reply src/utils/constants.ts:19]: Maybe we can just remove the required rule?

Comment (winniezheng) [reply src/utils/constants.ts:19]: I feel like in the future we should allow for "other". 
Yeah we could but also having the none of the above option might ensure that they don't have any of the listed allergens...safer for legal purposes maybe idk lol xD?

Comment (chrisrabe) [reply src/utils/constants.ts:19]: Yep agreed. I guess we could just have this at the moment and I'll create a ticket in our backlog about adding the Other option

**Review by chrisrabe — APPROVED** [review]
Will create a ticket about the Other option

**Pull request merged**